docs/detections/prebuilt-rules/rule-details/linux-restricted-shell-breakout-via-cpulimit-shell-evasion.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[linux-restricted-shell-breakout-via-cpulimit-shell-evasion]]
+=== Linux Restricted Shell Breakout via cpulimit Shell Evasion
+
+Identifies Linux binary cpulimit abuse to break out from restricted environments by spawning an interactive system shell. The cpulimit utility is used to restrict the CPU usage of a process in cases of CPU or system load exceeding the defined threshold and the activity of spawning a shell is not a standard use of this binary by a user or system administrator. This can potentially indicate a malicious actor attempting to improve the capabilities or stability of their access.
```

Comment:
```suggestion
Identifies abuse of the Linux binary cpulimit to break out from restricted environments by spawning an interactive system shell. The cpulimit utility is used to restrict the CPU usage of a process in case of CPU or system load that exceeds the defined threshold, and the activity of spawning a shell is not a standard use of this binary by a user or system administrator. This can potentially indicate a malicious actor attempting to improve the capabilities or stability of their access.
```